test/performance/logs/parameters.yml

```diff
@@ -0,0 +1,72 @@
+receivers: "logs"
+
+test_case: "logs_performance"
```

Comment:
I wonder if there is a way to have code parameters instead of using yaml. It is fine if you think is cleaner.